### ISSUE TITLE
Add a way for users to delete their account

### ISSUE THREAD

**Issue by Lesterpig** [openfun/fun-apps#495]
There is no way for users to delete their account. It could be really interesting, as users have a rectification / deletion right in many countries.

**Comment by rmoch** [comment]
Access to personal information, rectification or account deletion requests are processed by our student support team for now at contact@france-universite-numerique-mooc.fr

**Comment by Lesterpig** [comment]
Hi @rmoch, and thanks for your fast reply.

I've just opened this issue to purpose this as a _feature request_. Is there any problem to automatically remove users wanting to delete their accounts? It is far more efficient than manual processing (I believe).

Thank you again for your work!

**Comment by rmoch** [comment]
You are right. We think about giving this ability to our user as we like to have it on other sites. But you know so many things to do... Would you contribute ?
